_std/defines/speech_defines/speech.dm

```diff
@@ -1,3 +1,12 @@
+//------------- WRAPPERS -------------//
+/// A wrapper for _AddOutput that permits the usage of named arguments.
+#define AddOutput(output_id, arguments...) _AddOutput(output_id, list(##arguments))
+/// A wrapper for _AddInput that permits the usage of named arguments.
+#define AddInput(input_id, arguments...) _AddInput(input_id, list(##arguments))
+/// A wrapper for _AddModifier that permits the usage of named arguments. This caters to both speech and listen trees.
+#define AddModifier(modifier_id, arguments...) _AddModifier(modifier_id, list(##arguments))
```

Comment:
These names might be a bit generic? Could be worth adding `Speech` in front of them or similar

Reply:
They're used in the same context as the former procs; `AddOutput` shouldn't be used outside of `say_tree.AddOutput`.

Reply:
More so commenting how they pollute the global namespace for auto complete

Reply:
Good point, I'll push a change shortly.